### PR TITLE
ARCH-559 Display default avatar if user does not have a profile image.

### PR DESCRIPTION
**Pull request by douglashall**
See it in action at https://profile.edxdev.org/u/verified

### CODE REVIEW

File: src/components/ProfilePage/ProfileAvatar.jsx [hunk 1]
```diff
@@ -117,17 +118,25 @@ class ProfileAvatar extends React.Component {
...
   }
 
+  renderAvatar() {
+    return this.props.isDefault ? (
+      <DefaultAvatar className="text-muted" role="img" aria-hidden focusable="false" viewBox="0 0 24 24" />
+    ) : (
+      <img
+        className="w-100 h-100 d-block rounded-circle overflow-hidden"
+        style={{ objectFit: 'cover' }}
+        alt="profile avatar"
```
Comment (robrap) [inline src/components/ProfilePage/ProfileAvatar.jsx]: missing i18n.  I know this wasn't your fault, but maybe you could fix?

File: src/assets/avatar.svg [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 53.2 (72643) - https://sketchapp.com -->
+    <title>avatar</title>
+    <desc>Created with Sketch.</desc>
+    <g id="avatar" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <path d="M4.10255106,18.1351061 C4.7170266,16.0581859 8.01891846,14.4720277 12,14.4720277 C15.9810815,14.4720277 19.2829734,16.0581859 19.8974489,18.1351061 C21.215206,16.4412566 22,14.3122775 22,12 C22,6.4771525 17.5228475,2 12,2 C6.4771525,2 2,6.4771525 2,12 C2,14.3122775 2.78479405,16.4412566 4.10255106,18.1351061 Z M12,24 C5.372583,24 0,18.627417 0,12 C0,5.372583 5.372583,0 12,0 C18.627417,0 24,5.372583 24,12 C24,18.627417 18.627417,24 12,24 Z M12,13 C9.790861,13 8,11.209139 8,9 C8,6.790861 9.790861,5 12,5 C14.209139,5 16,6.790861 16,9 C16,11.209139 14.209139,13 12,13 Z" fill="currentColor"></path>
+    </g>
+</svg>
```
Comment (robrap) [inline src/assets/avatar.svg]: Can you add the newline?  Can we add https://eslint.org/docs/rules/eol-last to our linter?

File: config/webpack.dev.config.js [hunk 3]
```diff
@@ -64,9 +64,13 @@ module.exports = Merge.smart(commonConfig, {
       // bundles. This makes the javascript bundles ginormous and defeats caching so we will use the
       // file-loader instead to copy the files directly to the output directory.
       {
-        test: /\.(woff2?|ttf|svg|eot)(\?v=\d+\.\d+\.\d+)?$/,
+        test: /\.(woff2?|ttf|eot)(\?v=\d+\.\d+\.\d+)?$/,
         loader: 'file-loader',
       },
+      {
+        test: /.svg$/,
+        use: ['@svgr/webpack', 'url-loader'],
```
Comment (robrap) [inline config/webpack.dev.config.js]: Curious what this is all about. :)

Comment (douglashall) [reply config/webpack.dev.config.js]: This is the webpack loader for `svg` files.

**Pull request merged**